playwright.electron.config.ts

```diff
@@ -14,8 +14,8 @@ export default defineConfig({
   forbidOnly: true,
   /* Do not retry */
   retries: 0,
-  /* Different amount of parallelism on CI and local. */
-  workers: platform() === 'win32' ? 1 : 2,
+  /* Use all available CPU cores */
+  workers: process.env.CI ? '100%' : 1,
```

Comment:
Nice. I did bring it down from 8 to 2 back a few weeks ago as it didn't seem to lead to any slowdown at the time and allowed us to run cheaper runners, but I definitely like the idea of 100% and letting pw figure it out!